spring-cloud-circuitbreaker-hystrix/src/test/java/org/springframework/cloud/circuitbreaker/hystrix/HystrixCircuitBreakerIntegrationTest.java

```diff
@@ -53,12 +53,13 @@
 	protected static class Application {
 		@RequestMapping("/slow")
 		public String slow() throws InterruptedException {
-			Thread.sleep(3000);
```

Comment:
I dont understand the need for this.  It sleeps for 3s but hystrix is configured to time out in 2s.

Reply:
I've decreased the sleeping time to reduce total test time.
2s timeout does not need to sleep 3s.